purchase_reception_notify/__manifest__.py

```diff
@@ -1,11 +1,11 @@
-# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# Copyright 2019 ForgeFlow S.L.
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3.0).
 
 {
     "name": "Purchase Reception Notify",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "category": "Purchase Management",
-    "author": "Eficent, " "Odoo Community Association (OCA)",
+    "author": "ForgeFlow, " "Odoo Community Association (OCA)",
```

Comment:
make it one single String: `"blabla " "another blabla"` ->  `"blabla another blabla"`